Fix facebook marketing source SAT fail on invalid config file (#5621)
airbytehq/airbyte: airbyte-cdk/python/CHANGELOG.md

# Changelog

## 0.1.13

> In case of passed config file are not comply with spec schema raise an exception instead of sending airbyte error message and exit.

Reviewer: ```suggestion
If the input config file does not comply with spec schema, raise an exception instead of `system.exit`.
```
